src/object-identifier.scss

```diff
@@ -0,0 +1,50 @@
+@import "./new-settings";
+@import "./mixins";
+
+$block: #{$fd-namespace}-object-identifier;
+
+.#{$block} {
+  $fd-object-identifier-title-font-size: var(--sapFontLargeSize);
+  $fd-object-identifier-title-font-weight: bold;
+
+  &__title {
+    @include fd-reset();
+
+    color: var(--sapList_TextColor);
+    font-size: $fd-object-identifier-title-font-size;
+
+    &--link {
+      @include fd-link();
+      @include action-cursor();
+    }
+
+    &--bold {
+      font-weight: $fd-object-identifier-title-font-weight;
+
+      .fd-link {
+        font-size: $fd-object-identifier-title-font-size;
+        font-weight: $fd-object-identifier-title-font-weight;
+      }
+    }
+
+    .fd-link {
+      font-size: $fd-object-identifier-title-font-size;
+    }
```

Comment:
This way you are violating the `self-contained` rules. You are modifying the styling of an element inside another element. If this needs to be done you need another class. Let's say you need to change the font-size of the `fd-link` in this case:
```
<p class="fd-object-identifier__title"><a href="#" class="fd-link">NoteBook Basic 15</a></p>
```

You should not do `.fd-object-identifier__title .fd-link`, you need to create a class `fd-object-identifier__link` and apply it together with `.fd-link` and basically rely on the specificity. 

You will have:
```
<p class="fd-object-identifier__title"><a href="#" class="fd-link fd-object-identifier__link">NoteBook Basic 15</a></p>
```

and then:
```
.fd-object-identifier__link {
    font-size: 20px;
}
```

Reply:
fixed